RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -35,7 +35,7 @@
 trackingPhase2PU140.toModify(highPtTripletStepSeedLayers,
 # combination with gap removed as only source of fakes in current geometry (kept for doc) 
     layerList = ['BPix1+BPix2+BPix3', 'BPix2+BPix3+BPix4',
-#                 'BPix1+BPix3+BPix4', 'BPix1+BPix2+BPix4',
+                 'BPix1+BPix3+BPix4', 'BPix1+BPix2+BPix4',
```

Comment:
is either of these two more useful than the other?
Given the large increase in fake rate, maybe some  optimization can be done